packages/template-retail-react-app/app/components/shared/ui/index.jsx

```diff
@@ -0,0 +1,107 @@
+export {Divider} from './Divider'
+export {Drawer} from './Drawer'
+export {DrawerBody} from './DrawerBody'
+export {DrawerCloseButton} from './DrawerCloseButton'
+export {DrawerContent} from './DrawerContent'
+export {DrawerFooter} from './DrawerFooter'
+export {DrawerHeader} from './DrawerHeader'
+export {DrawerOverlay} from './DrawerOverlay'
+export {Fade} from './Fade'
+export {Flex} from './Flex'
+export {FormControl} from './FormControl'
+export {FormErrorMessage} from './FormErrorMessage'
+export {FormLabel} from './FormLabel'
+export {Grid} from './Grid'
+export {GridItem} from './GridItem'
+export {HStack} from './HStack'
+export {Heading} from './Heading'
+export {Icon} from './Icon'
+export {IconButton} from './IconButton'
+export {Image} from './Image'
+export {Img} from './Img'
+export {Input} from './Input'
+export {InputGroup} from './InputGroup'
+export {InputLeftElement} from './InputLeftElement'
+export {InputRightElement} from './InputRightElement'
+export {Link} from './Link'
+export {List} from './List'
+export {ListItem} from './ListItem'
+export {Modal} from './Modal'
+export {ModalBody} from './ModalBody'
+export {ModalCloseButton} from './ModalCloseButton'
+export {ModalContent} from './ModalContent'
+export {ModalFooter} from './ModalFooter'
+export {ModalHeader} from './ModalHeader'
+export {ModalOverlay} from './ModalOverlay'
+export {Popover} from './Popover'
+export {PopoverArrow} from './PopoverArrow'
+export {PopoverBody} from './PopoverBody'
+export {PopoverCloseButton} from './PopoverCloseButton'
+export {PopoverContent} from './PopoverContent'
+export {PopoverFooter} from './PopoverFooter'
+export {PopoverHeader} from './PopoverHeader'
+export {PopoverTrigger} from './PopoverTrigger'
+export {Portal} from './Portal'
+export {Radio} from './Radio'
+export {RadioGroup} from './RadioGroup'
+export {Select} from './Select'
+export {SimpleGrid} from './SimpleGrid'
+export {Skeleton} from './Skeleton'
+export {Spacer} from './Spacer'
+export {Spinner} from './Spinner'
+export {Stack} from './Stack'
+export {StackDivider} from './StackDivider'
+export {StylesProvider} from './StylesProvider'
+export {Text} from './Text'
+export {Tooltip} from './Tooltip'
+export {VStack} from './VStack'
+export {Wrap} from './Wrap'
+export {WrapItem} from './WrapItem'
+export {createStylesContext} from './createStylesContext'
+export {extendTheme} from './extendTheme'
+export {useBreakpoint} from './useBreakpoint'
+export {useBreakpointValue} from './useBreakpointValue'
+export {useDisclosure} from './useDisclosure'
+export {useMediaQuery} from './useMediaQuery'
+export {useMultiStyleConfig} from './useMultiStyleConfig'
+export {useNumberInput} from './useNumberInput'
+export {useRadio} from './useRadio'
+export {useRadioGroup} from './useRadioGroup'
+export {useStyleConfig} from './useStyleConfig'
+export {useStyles} from './useStyles'
+export {useTheme} from './useTheme'
+export {useToast} from './useToast'
```

Comment:
This is one of two files that demonstrates the approach. All imports like 

```
import {foo} from '@chakra-ui/react'
```
 will now instead import like this
 
 ```
import {foo} from '@salesforce/retail-react-app/app/components/shared/ui'
```

Outside of three directory index files 
- `app/components/shared/ui` 
- `app/components/shared/util` 
- `app/components/shared/theme`

The only changes are just a find / replace in the whole project for `@chakra-ui/react` => `@salesforce/retail-react-app/app/components/shared/ui` (and so on for others)